test.sh

```diff
@@ -16,3 +16,6 @@ fgt golint ./...
 go test ./...
 go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' ./... | xargs -i sh -c {}
 gover . coverprofile.txt
+go get -d ./...
+go build -tags nopkcs11 cmd/...
```

Comment:
Why only build cmd with the nopkcs11 tag?

Also FYI: depending on how long this takes, you could speed up the total time your tests take with travis' matrix feature. E.g. you'd replace `go test ./...` with `go test $BUILD_FLAGS ./...` and put in your .travis.yml:

```
env:
  matrix:
    - BUILD_FLAGS=
    - BUILD_FLAGS="-tags nopkcs11"
```